frontend/packages/dev-console/src/components/topology/TopologyPage.scss

```diff
@@ -7,6 +7,27 @@
   top: 0;
   bottom: 0;
 
+  &__toolbar-hints {
+    align-items: center;
+    background: white;
+    border: 1px solid var(--pf-global--BorderColor--light-100);
+    border-radius: 8px;
+    display: flex;
+    padding: var(--pf-global--spacer--xs) var(--pf-global--spacer--sm);
+    position: absolute;
+    top: var(--pf-global--spacer--sm);
+    z-index: 2;
+    &__icon {
+      color: var(--pf-global--info-color--100);
+    }
+    &__text {
```

Comment:
Can we add some more padding to the right of the `i` icon:
![image](https://user-images.githubusercontent.com/14068621/77363573-5a0b5400-6d29-11ea-93dc-f0fd59f21472.png)

Reply:
![image](https://user-images.githubusercontent.com/11633780/77565196-3f0c2180-6e9a-11ea-81de-b21300cb9f02.png)